setup.py

```diff
@@ -1,14 +1,19 @@
 #!/usr/bin/env python
-from setuptools import setup
+from setuptools import find_packages, setup
 import healthcheck
 
+with open('requirements.txt') as requirements_file:
+    test_requirements = requirements_file.readlines()
+
 setup(
     name='healthcheck',
     version=healthcheck.__version__,
     description=healthcheck.__doc__,
     author='Yola',
     author_email='engineers@yola.com',
+    license='MIT (Expat)',
     url=healthcheck.__url__,
-    packages=['healthcheck'],
+    packages=find_packages(exclude=('tests', '*.tests')),
+    setup_requires=['nose < 2.0.0'],
+    tests_require=test_requirements,
```

Comment:
Ah, yes, there is that. `setup.py test` is the standard, though. So we should make it work.